outreaches.md

```diff
@@ -0,0 +1,496 @@
+    * [Related Objects](#related-objects)
+    * [Links](#links)
+* [Helpers](#helpers)
+* [Related Resources](#related-resources)
+* [Scenarios](#scenarios)
+    * [Scenario: Retrieving a collection of Outreach resources (GET)](#scenario-retrieving-a-collection-of-outreach-resources-get)
+    * [Scenario: Retrieving an individual Outreach resource (GET)](#scenario-scenario-retrieving-an-individual-outreach-resource-get)
+    * [Scenario: Creating a new outreach (POST)](#scenario-creating-a-new-outreach-post)
+    * [Scenario: Modifying an outreach (PUT)](#scenario-modifying-an-outreach-put)
+    * [Scenario: Deleting an outreach (DELETE)](#scenario-deleting-an-outreach-delete)
+
+
+{% include endpoints_and_url_structures.md %}
+
+The link relation label for a Outreach resource is ```osdi:outreach``` for a single Outreach resource or ```osdi:outreaches``` for a collection of Outreach resources.
+
+_[Back to top...](#)_
+
+
+## Fields
+
+{% include fields_intro.md %}
+
+{% include global_fields.md %}
+
+_[Back to top...](#)_
+
+
+### Outreach Fields
+
+A list of fields specific to the Outreach resource.
+
+| Name          | Type      | Description
+|-----------    |-----------|-----------|--------------
+|origin_system		|string     |A human readable identifier of the system where this outreach was created. (ex: "OSDI System")
+|action_date		|string		|The date and time the outreach was made by the person.
+|type				|flexunum	|The type of outreach, specifying how the user performed the outreach to targets. One of "email", "in-person", "phone", "postal mail", or another type as needed.
+|duration		|integer		|The duration in seconds of the outreach, if applicable. (ex: duration will only be present on phone outreach types)
+|subject		|string		|The subject of the outreach, if applicable. (ex: subject will only be present on email outreach types)
+|message		|string		|The message of the outreach, if applicable. (ex: message will only be present on email or postal mail outreach types)
+|targets			|[Target[]](#target)    |An array of target object hashes representing the targets of the outreach.
+
+_[Back to top...](#)_
+
+
+### Related Objects
+
+These JSON hashes included in the table above are broken out into their own tables for readability, rather than independent resources with their own endpoints.
+
+#### Target
+
+|Name          	|Type      |Description
+|-----------    |-----------|--------------
+|target.title	|string    |The title of the target. (ex: "Senator")
+|target.given_name	|string    |The first or given name of the target. (ex: "John")
+|target.family_name	|string    |The last or family name of the target. (ex: "Smith")
+|target.ocdid	|string    |The Open Civic Data Division ID for this target's political geography, if applicable. See [here](http://docs.opencivicdata.org/en/latest/proposals/0002.html) for more documentation. (ex: "ocd-division/country:us/state:ny/cd:18", which corresponds to New York's 18th Congressional District)

```

Comment:
Yeah NVM about email and phone number, I see your point there.

I see your point about title.  I think I glossed right over that because I assumed it was meant as an honorific ("Mr." / "Dr.", etc.) whereas what I think will matter really is "which position within the organization did we target" - was it the VP of HR?  The CEO?  The Agriculture Commissioner? etc.